parts/linux/cloud-init/artifacts/sync-tunnel-logs.sh

```diff
@@ -39,7 +39,7 @@ while true; do
 done &
 
 # Manually sync all matching logs once
-for TUNNEL_LOG_FILE in $(compgen -G "$SRC/@(aks-link|azure-cns|cilium|konnectivity|tunnelfront)-*_kube-system_*.log"); do
+for TUNNEL_LOG_FILE in $(compgen -G "$SRC/*_kube-system_*.log"); do
```

Comment:
TIL. if I understand, this is a bash built-in, so fine on mariner?

Reply:
yeah, it's part of bash, and is a good way to expand the glob.